### DISCUSSION TITLE
How to move a window to Display 4

### DISCUSSION THREAD

**Discussion by cooper7012** [rxhanson/RectanglePro-Community#539] [General]
How do I get a window to move to a position on display 4 (see screenshot)
![Screenshot 2024-07-30 at 5 11 05 PM](https://github.com/user-attachments/assets/25885b10-15ab-432a-8bb6-22be90708d6a)

**Comment by Aaron-Rumpler** [comment]
You can add a keyboard shortcut to move the current window to the previous or next space, but that's the extent of the support for spaces in Rectangle Pro. There's a comment from the developer (@rxhanson) [here](https://github.com/rxhanson/RectanglePro-Community/discussions/435#discussioncomment-8263347), but basically macOS doesn't provide a good way for Rectangle Pro to work with spaces.
